public/index.php

```diff
@@ -16,6 +16,14 @@
 	));
 });
 
+$klein->respond('/create', function() use ($twig, $crud){
+	$crud->create($_POST['name'], $_POST['surname'], $_POST['indexno'], $_POST['address']);
+	$getAll = $crud->getAll();
+	echo $twig->render('home.twig', array(
```

Comment:
instead all of this, you can simply let the user know that 'student' has been created - if operation was successful, or if not, what went wrong